repl/src/test/scala/ammonite/repl/EulerTests.scala

```diff
@@ -317,8 +317,6 @@ object EulerTests extends TestSuite{
                    |53503534226472524250874054075591789781264330331690""".stripMargin.replace("\n", " ")
       check.session(s"""
         @ val s = "$data"
-        s: java.lang.String = "$data"
```

Comment:
I think you need to leave a blank line here for the `check.session` function to treat these as separate commands. Otherwise it expects the first command to be incomplete (`Buffer`)